app/controllers/medicaid/health_introduction_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Medicaid
+  class HealthIntroductionController < ApplicationController
```

Comment:
@disambiguator I inherited from `ApplicationController` and only included methods required for a step to work. If we are happy with this, I can update the other breather controllers

Reply:
If I'm reading this right - would the other breather controllers need everything below this comment?

Reply:
yep! every controller for a breather page would need all of these methods

Reply:
where's the :shrug: emoji when you need it!

Reply:
may be helpful to compare with the controller changes required to add a breather page *with* inheritance https://github.com/codeforamerica/michigan-benefits/pull/481/files#diff-a1c0b16dd0b1db3ce5d021c461d4edba

Reply:
> yep! every controller for a breather page would need all of these methods

That much duplication, in my opinion, is not ideal. But I'm not entirely sure what the best / right answer is to combat this outside of inheritance or another more composition-based approach. The latter of which I think would need some further discussion with the whole group